src/services/search.ts

```diff
@@ -0,0 +1,45 @@
+import uFuzzy from '@leeoniya/ufuzzy';
+import { debounce as debounceLodash } from 'lodash';
+
+const uf = new uFuzzy({
+  intraMode: 1,
+  intraIns: 1,
+  intraSub: 1,
+  intraTrn: 1,
+  intraDel: 1,
+});
+
+export function fuzzySearch(haystack: string[], query: string, dispatcher: (data: string[][]) => void) {
```

Comment:
Dispatcher is an unexpected name for a callback in this context.

Reply:
Fixed in https://github.com/grafana/explore-logs/pull/434/commits/a2943516059c3ad56c6032eaf03fb1ea0815e40d